xlsxtream.gemspec

```diff
@@ -20,10 +20,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
   spec.required_ruby_version = '>= 1.9.1'
 
-  spec.add_dependency "rubyzip", ">= 1.0.0"
+  spec.add_dependency "zip_tricks", "~> 4.5"
 
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency 'rubyzip', '~> 1.1'
```

Comment:
Use double quotes for consitency and `~> 1.2` to match changes from master.

Reply:
Do you want rubyzip as a development dependency still, zip_tricks as a development dependency too, or both as runtime dependencies?

Reply:
You can keep rubyzip as development dependency, I'll remove it as soon as I've verified everything is fine with zip_tricks.